working-release-notes/template.md

```diff
@@ -0,0 +1,11 @@
+# Feature Name (remember to update the file name to match)
+
+authors = ["@cart", "@Jondolf"]
+contributors = ["@alice-i-cecile", "@killercup", "@BD103", "@fluffiac"]
+prs = [14791, 15458, 15269]
```

Comment:
Damn, I made it in the template 👀 haha

Do we want to keep it like this, or use some "fake" names for the authors and contributors? Like `[@CoolBevyContributor, @ferris-the-crab]` or something.

Reply:
Yeah let's swap the names to something fake.